src/game/object_helpers.c

```diff
@@ -129,21 +129,23 @@ Gfx *geo_switch_area(s32 callContext, struct GraphNode *node, UNUSED void *conte
 #ifdef ENABLE_VANILLA_LEVEL_SPECIFIC_CHECKS
             if (gCurrLevelNum == LEVEL_BBH) {
                 // In BBH, check for a floor manually, since there is an intangible floor. In custom hacks this can be removed.
```

Comment:
not requesting changes here, but we really should just add a flag to surface load if an area loads an intangible floor